api-types/lib/index.d.ts

```diff
@@ -63,5 +63,7 @@ export type DesktopAPI = {
     onOpenThreadForCalls: (listener: (threadID: string) => void) => () => void;
     openStopRecordingModal: (channelID: string) => void;
     onOpenStopRecordingModal: (listener: (channelID: string) => void) => () => void;
+    openCallsUserSettings: () => void;
+    onOpenCallsUserSettings: (listener: () => void) => () => void;
```

Comment:
@streamer45 Can you run `npm run build` and increment the `api-types` version as part of this PR?

Reply:
@devinbinnie Done in https://github.com/mattermost/desktop/pull/3109/commits/0353d98a0381319a15cf3b6d60439d06d116a87f

Reply:
I also just realized, these are missing from `api-types/index.ts`, which is what we should be editing. Can we add them there?

Reply:
Yeah, got confused with all these files.